index.html

```diff
@@ -8,70 +8,64 @@
 	<link rel="stylesheet" href="https://dev.mavo.io/dist/mavo.css">
 	<link rel="stylesheet" href="assets/css/style.css">
 </head>
-<body>
+<body mv-app="dpi" class="mv-autoedit" mv-storage="https://github.com/LeaVerou/dpi/blob/gh-pages/devices.json" mv-bar="no-login">
 	<header>
 		<h1>
 			<img src="assets/img/logo.svg" />
 			<strong>dpi</strong>love
 		</h1>
 	</header>
-
-	<fieldset>
-		Resolution:
-		<input id="width" inputmode="numeric" pattern="\d+" />
-		×
-		<input id="height" inputmode="numeric" pattern="\d+" />
-
-		<p id="resolutions">Common:
-			<a href="#" tabindex="-1">3840×2160</a>
-			<a href="#" tabindex="-1">2560×1440</a>
-			<a href="#" tabindex="-1">1920×1080</a>
-			<a href="#" tabindex="-1">1680×1050</a>
-			<a href="#" tabindex="-1">1440×900</a>
-			<a href="#" tabindex="-1">1366×768</a>
-			<a href="#" tabindex="-1">1280×800</a>
-			<a href="#" tabindex="-1">1024×768</a>
-		</p>
-	</fieldset>
-
-	<fieldset>
-		<select id="dimension" title="Physical dimension (in inches)">
-			<option selected value="d">Diagonal</option>
-			<option value="w">Width</option>
-			<option value="h">Height</option>
-		</select>:
-		<input id="physical" value="13.3" inputmode="numeric" pattern="\d*\.?\d+" />″
-
-		<p id="diagonals">Common diagonals:
-			<a href="#" tabindex="-1">7″</a>
-			<a href="#" tabindex="-1">11.6″</a>
-			<a href="#" tabindex="-1">13.3″</a>
-			<a href="#" tabindex="-1">14″</a>
-			<a href="#" tabindex="-1">15.6″</a>
-			<a href="#" tabindex="-1">17.3″</a>
-			<a href="#" tabindex="-1">21″</a>
-			<a href="#" tabindex="-1">24″</a>
-			<a href="#" tabindex="-1">27″</a>
-		</p>
-	</fieldset>
-
-	<output>
-		<div>
-			<strong id="result"></strong>
-			pixels per inch
-		</div>
-	</output>
+	<div class="mv-bar mv-ui" mv-bar="no-login"></div>
+	<section>
+		<fieldset>
+			Resolution:
+			<input id="width" property inputmode="numeric" pattern="\d+" class="mv-autosize" mv-storage="none">
+			×
+			<input id="height" property inputmode="numeric" pattern="\d+" class="mv-autosize" mv-storage="none">
+			<meta property="ratio" content="[width/height]">
+			<p id="resolutions">Common:
+				<a mv-multiple="resolution" href="#[w]×[h]" tabindex="-1" mv-action="set(width, w) set(height, h)" mv-attribute="none">
+					<span property="w"></span>×<span property="h"></span>
+				</a>
+			</p>
+		</fieldset>
+		
+		<fieldset>
+			<select id="dimension" property title="Physical dimension (in inches)">
+				<option selected value="d">Diagonal</option>
+				<option value="wide">Width</option>
+				<option value="high">Height</option>
```

Comment:
Just noticed, why were these values changed?

Reply:
This happens when the values are the same as property names. 
<img width="782" alt="screen shot 2018-06-12 at 2 45 44 pm" src="https://user-images.githubusercontent.com/16854657/41310419-74106000-6e4f-11e8-95da-5cd822081ad0.png">
Maybe I should change the property names instead for consistency?

Reply:
Can you investigate why? You have the inspector, you can select that element and try property references and see what they resolve to.

Looking quickly at your expression, it looks like you are using unquoted text, which is flimsy: If you have properties with those names, these references (e.g. `d`) will be the values of the properties, not the literal text you intended. It's a good practice to always use quotes to prevent surprises. This is likely related to the issues you're having, and probably also the cause of the weirdness when you use `d` etc as property names.